html-block-and-inline/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>
+    <div>Price: <s>$19.99</s> $14.99</div><br/>
+    <div>Part of our new Americana collection, this <em>beatuiful</em> blue is alrady a customer favorite!</div><br/>
+    <div>Enjoy complete <strong>Freedom of Movement!</strong></div><br/>
+    <div>Does not slip and slide like a regular blanket and keeps you warm from head to toe.</div><br/>
```

Comment:
`div` elements should not be used as text elements. If you want normal-sized default text, you should just use a `p` element. Also, the `br` elements are unnecessary, you should remove them.

Reply:
Just fixed, thanks!